.bcr/metadata.template.json

```diff
@@ -0,0 +1,13 @@
+{
+  "homepage": "https://github.com/bazel-contrib/rules_cuda",
```

Comment:
Should this be rules_foreign_cc rather than rules_cuda?

Reply:
```suggestion
  "homepage": "https://github.com/bazel-contrib/rules_foreign_cc",
```